### PR TITLE
updated c++ codes (string_to_uppercase_and_lowercase)

### PR DESCRIPTION
**Pull request by dhruvdy**
Issue #656

### CODE REVIEW

**Comment by OjusWiZard** [comment]
To which issue does this belong?

**Comment by dhruvdy** [comment]
#656

File: Code/C++/string_to_lowercase_and_uppercase.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+       cout<<"Lowercase ->"<<lowercase_string<<endl;
+       cout<<"Uppercase ->"<<uppercase_string<<endl;
+       return 0;
+   }
+
+
+   /*
+
+   Time complexity O(n)
```
Comment (rudrakshi99) [inline Code/C++/string_to_lowercase_and_uppercase.cpp:42]: add complexities to the end.

File: Code/C++/string_to_lowercase_and_uppercase.cpp [hunk 2]
```diff
@@ -0,0 +1,58 @@
+/*program to convert string to lowercase and uppercase*/
+#include<iostream>
+#include<string.h>
```
Comment (rudrakshi99) [inline Code/C++/string_to_lowercase_and_uppercase.cpp:3]: use only `string`

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by dhruvdy** [comment]
Can one of the admins verify the code ?
I made the changes as requested.

**Review by rudrakshi99 — APPROVED** [review]

**Comment by rudrakshi99** [comment]
@dhruvdy update the list in the string folder.

**Comment by dhruvdy** [comment]
@rudrakshi99 I have updated the list in string folder.

**Comment by rudrakshi99** [comment]
please resolve merge conflicts then we are good to merge.

**Comment by dhruvdy** [comment]
@rudrakshi99 I am unable to make this conflicting file correct. Please help me.

**Comment by dhruvdy** [comment]
@rudrakshi99 please, can you check the readme file now. Is it fine?

File: Code/C++/string_to_lowercase_and_uppercase.cpp [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+string lowercase(string s1){
+   for(int i=0;i<s1.size();++i)
+   {
+    if(s1[i]>=65 && s1[i]<=90)
+            s1[i]=s1[i]+32;//conversion to lowercase
+    }
+    return s1;
+
```
Comment (OjusWiZard) [inline Code/C++/string_to_lowercase_and_uppercase.cpp:14]: Remove this line

**Review by OjusWiZard — CHANGES_REQUESTED** [review]
Maintain the indentation of 4 spaces throughout the code.

**Comment by dhruvdy** [comment]
@OjusWiZard I made the changes as said. Please review the code and kindly merge it if everything is fine now.

File: Code/C++/string_to_lowercase_and_uppercase.cpp [hunk 4]
```diff
@@ -1,39 +1,38 @@
...
-
-string uppercase(string s1){
-   for(int i=0;i<s1.size();++i)
-    if(s1[i]>=97 && s1[i]<=122)
-    {
-        s1[i]=s1[i]-32;//convertion to uppercase case
-    }
-    return s1;
+      for(int i=0;i<s1.size();++i)
+       if(s1[i]>=97 && s1[i]<=122)
```
Comment (rudrakshi99) [inline Code/C++/string_to_lowercase_and_uppercase.cpp:20]: use curly brackets for `for loop`

**Comment by dhruvdy** [comment]
@rudrakshi99 I did it. Please check if some other changes are to be done..

**Review by rudrakshi99 — APPROVED** [review]

**Pull request merged**